scripts/bootloader/keygen.py

```diff
@@ -8,28 +8,35 @@
 from ecdsa import SigningKey
 from ecdsa.curves import NIST256p
 import argparse
+import sys
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Generate PEM file.",
         formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument("--output-private", required=False,
-                        help="Output private key file name.")
-    parser.add_argument("--output-public", required=False,
-                        help="Output public key file name.")
+    parser.add_argument("--private", required=False, action="count",
```

Comment:
Why `action="count"` ?

Reply:
Applies elsewhere as well.

Reply:
I copied from the verbose argument, but I guess `store_true` is better for this. Will change.

https://docs.python.org/3/library/argparse.html#action